### PR TITLE
Compatibility with absolute URLs (from a ZIM) in jQuery mode.

### PR DESCRIPTION
**Pull request by mossroy**
For example for wikipedia_fr_all_09_2012.zim

### CODE REVIEW

**Comment by mossroy** [comment]
This is finally not crucial, as the ZIM files generated since the end of 2014 all have relative URLs now

File: www/js/app.js [hunk 1]
```diff
@@ -955,9 +957,14 @@ define(['jquery', 'abstractBackend', 'util', 'cookies','geometry','osabstraction
                     // It's a link to another article
                     // Add an onclick event to go to this article
                     // instead of following the link
+                    
                     if (url.length>=2 && url.substring(0, 2) === "./") {
                         url = url.substring(2);
                     }
+                    // Remove the initial slash if it's an absolute URL
+                    else if (url.length>=1 && url.substring(0, 1) === "/") {
```
Comment (peter-x) [inline www/js/app.js]: Length check is not needed

File: www/js/app.js [hunk 2]
```diff
@@ -978,7 +985,11 @@ define(['jquery', 'abstractBackend', 'util', 'cookies','geometry','osabstraction
                     });
                 } else {
                     // It's a standard image contained in the ZIM file
-                    var imageMatch = image.attr("src").match(regexpImageUrl);
+                    // We try to find its name (from an absolute or relative URL)
+                    var imageMatch = image.attr("src").match(regexpImageAbsoluteUrl);
```
Comment (peter-x) [inline www/js/app.js]: We could use one regex for both cases.

**Comment by mossroy** [comment]
I implemented your suggestions and merged the pull request

**Pull request merged**